tests/check/tests.yaml

```diff
@@ -1975,21 +1975,6 @@ tests:
               user: a:b:c
             errorCode: 2000
 
-  - name: validation_circular_model
```

Comment:
Why are we removing this?

Reply:
Oh, is it because this is part of a model validation test instead?

Reply:
Because this test fails in the WriteAuthorizationModel step now, instead of the Check itself